pkg/create/templates/docker.go

```diff
@@ -11,8 +11,8 @@ FROM openshift/base-centos7
 # ENV BUILDER_VERSION 1.0
 
 # TODO: Set labels used in OpenShift to describe the builder image
-#LABEL k8s.io/description="Platform for building xyz" \
-#      k8s.io/display-name="builder x.y.z" \
+#LABEL io.k8s.description="Platform for building xyz" \
+#      io.k8s.display-name="builder x.y.z" \
```

Comment:
hmm... I thought k8s was still using kis.io/display-name, was this confirmed somewhere?